agent/fw_wordpress.c

```diff
@@ -481,18 +481,18 @@ NR_PHP_WRAPPER(nr_wordpress_exec_handle_tag) {
   NR_UNUSED_SPECIALFN;
   (void)wraprec;
 
-  NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_WORDPRESS);
+  if (nrlikely(0 != NRINI(wordpress_hooks))) {
```

Comment:
```suggestion
  if (0 != NRINI(wordpress_hooks)) {
```
I don't think we want to override the branch predictor here. Within a given run, this will always evaluate to the same true/false, so the branch predictor will be able to hit with 100% after a pass or so. Otherwise, we are always taking a hit when hooks are off.

Reply:
> we are always taking a hit when hooks are off.

@ZNeumann This line is more of an assert, than anything - this code path (`nr_wordpress_exec_handle_tag` function) will never execute if hooks instrumentation is disabled. It is guaranteed by this line: https://github.com/newrelic/newrelic-php-agent/blob/1fbb0d121afd80145c114beeac94f32ef6cf78a1/agent/fw_wordpress.c#L599
Are you saying it is still better not to give this hint?

Reply:
Ah, since the function is never executed in this case, it makes no difference to me whether this `nrlikely` exists